### ISSUE TITLE
Groups button should not show when on groups page

### ISSUE THREAD

**Issue by tacryt-socryp** [urbit/urbit#3276]
![5B6CF2AA-8B9F-4031-A63E-1EBEAE2F766F](https://user-images.githubusercontent.com/1377557/89859546-18533c80-db56-11ea-8311-9caef567f922.jpeg)

**Comment by matildepark** [comment]
@urcades thoughts?

**Comment by urcades** [comment]
Is this a mobile view? If so, the optimal way to approach this question lies in the total reconsideration of navigating Landscape on a mobile interface. I think we can do quite a lot to modify how our mobile interface is presented.

Otherwise,

My POV boils down to it being "OK" that the groups button remains a static nav element in the menubar area. 

I've always found it to be a bit weird that the "Home" button disappears while in the "Home context", given a structure like a menubar/top bar/header interface should remain 'foundation-like' – or static, basically – in its presentation. If our menubar is hiding and showing interactive elements all the time, that's just more 'state' our users need to hold in their heads, and it generally gives the interface less of a 'solid' presentation.

The conjoined consideration here is that at some point, Landscape owners should _really_ be able to modify their interfaces easily. People should be able to render their menubars as text-only, icon-only, in dark mode or light mode, etc. This effectively necessitates building the menubar as a framework of sorts at some point – an extendable/customizable interface people can make productive use of.

tl;dr: I think it's fine for the button to show while in groups, but this question points at a greater consideration for how we treat elements in the top bar in general, as a framework. Also, mobile nav UI needs to be rethought/optimized.

**Comment by tacryt-socryp** [comment]
Then why does the home button not display on the launch page? All good to not do hiding of buttons, but I want consistency!

**Comment by matildepark** [comment]
As per mock! As per mock! *shrinks into corncob*

**Comment by urcades** [comment]
Yes, sorry for not responding more quickly — this was probably observed in
the mock, but is a pattern I'd prefer reverting to one of
solidity/continuity (no disappearing) until we develop a framework as
described above for allowing users to customize, reduce, pin-to, or
disappear their menubars.

—
~fabled-faster
https://urbit.org/
